package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "postgrator",
-  "version": "4.0.0",
+  "version": "4.2.0",
```

Comment:
Looks like the source repo still has some small updates. I've postfixed versions with "-isight" in other areas where this happens to avoid version confusion. Perhaps we should also pull in some of those updates?